### PR TITLE
new test setup

### PR DESCRIPTION
**Pull request by ChrisRackauckas**
@kanav99

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
Looks like this caught an error introduced by the recent PR

**Comment by kanav99** [comment]
Sorry, I have made the fix in #109. We should rebase this PR after the merge

File: test/runtests.jl [hunk 1]
```diff
@@ -1,7 +1,30 @@
...
+  @time @safetestset "FiniteDiff Standard Tests" begin include("finitedifftests.jl") end
+  @time @safetestset "Color Differentiation Tests" begin include("coloring_tests.jl") end
+  @time @safetestset "Out of Place Tests" begin include("out_of_place_tests.jl") end
+end
+
+if GROUP == "All" || GROUP == "Downstream"
+  activate_downstream_env()
+  @time @safetestset "ODEs" begin
+    import OrdinaryDiffEq
+    include(joinpath(dirname(pathof(OrdinaryDiffEq)), "..", "test/interface/sparsedifftests.jl"))
```
Comment (kanav99) [inline test/runtests.jl:26]:
```suggestion
    include(joinpath(dirname(pathof(OrdinaryDiffEq)), "..", "test/interface/sparsediff_tests.jl"))
```

File: test/downstream/Project.toml [hunk 2]
```diff
@@ -0,0 +1 @@
+OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
```
Comment (kanav99) [inline test/downstream/Project.toml:1]:
```suggestion
[deps]
OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
```

**Pull request merged**